mods/ra/weapons/missiles.yaml

```diff
@@ -90,16 +89,15 @@ HellfireAG:
 		Inaccuracy: 128
 		Image: DRAGON
 		HorizontalRateOfTurn: 10
-		RangeLimit: 4c819
+		RangeLimit: 8c819
```

Comment:
0c1024 = 1c0. Feel free, as long as you document the percentages at least in the PR or the commit message.